server/notification-providers/dingding.js

```diff
@@ -11,14 +11,18 @@ class DingDing extends NotificationProvider {
      */
     async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
         const okMsg = "Sent Successfully.";
-
+            // Convert isAtAll to Boolean
+            notification.isAtAll = this.convertToBoolean(notification.isAtAll);
         try {
             if (heartbeatJSON != null) {
                 let params = {
                     msgtype: "markdown",
                     markdown: {
                         title: `[${this.statusToString(heartbeatJSON["status"])}] ${monitorJSON["name"]}`,
                         text: `## [${this.statusToString(heartbeatJSON["status"])}] ${monitorJSON["name"]} \n> ${heartbeatJSON["msg"]}\n> Time (${heartbeatJSON["timezone"]}): ${heartbeatJSON["localDateTime"]}`,
+                    },
+                    "at": {
+                        "isAtAll": notification.isAtAll // Here the isAtAll value in the notification object is used
```

Comment:
comment does not add valuable context => remove

```suggestion
                        "isAtAll": notification.isAtAll,
```